app/src/main/java/com/macbitsgoa/events/timer/TimerCardFragment.java

```diff
@@ -0,0 +1,86 @@
+package com.macbitsgoa.events.timer;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.os.CountDownTimer;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.google.android.material.card.MaterialCardView;
+import com.macbitsgoa.events.R;
+import com.macbitsgoa.events.home.HomeCardInterface;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import androidx.fragment.app.Fragment;
+
+/**
+ * The fragment for feature eateries to be shown on home screen
+ * {@link com.macbitsgoa.events.home.HomeActivity}.
+ */
+public class TimerCardFragment extends Fragment
+        implements HomeCardInterface {
+
+
+
+    @Override
+    public View onCreateView(final LayoutInflater inflater,
+                             final ViewGroup container, final Bundle savedInstanceState) {
+        final MaterialCardView selfCard =
+                (MaterialCardView) inflater.inflate(R.layout.fragment_timer_card,
+                        container, false);
+        final TextView tv= (TextView) selfCard.findViewById(R.id.timer_text);
+
+        int daysToGo =1;
```

Comment:
do not hardcode the values

use those defined in **app/build.gradle**, for example, here you can use the string `startDate`